[2022-03-11 18:52:44,198] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_01 scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2022-03-11 18:52:44,396] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_01 scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2022-03-11 18:52:44,397] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-11 18:52:44,397] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-03-11 18:52:44,397] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-11 18:52:45,004] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): task_01> on 2022-03-10 00:00:00+00:00
[2022-03-11 18:52:45,180] {standard_task_runner.py:52} INFO - Started process 5221 to run task
[2022-03-11 18:52:45,284] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_01', 'scheduled__2022-03-10T00:00:00+00:00', '--job-id', '71', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dags.py', '--cfg-path', '/tmp/tmpw5lymzd0', '--error-file', '/tmp/tmp_vdwsa5s']
[2022-03-11 18:52:45,305] {standard_task_runner.py:77} INFO - Job 71: Subtask task_01
[2022-03-11 18:52:47,355] {logging_mixin.py:109} INFO - Running <TaskInstance: parallel_dag.task_01 scheduled__2022-03-10T00:00:00+00:00 [running]> on host airflowvm
[2022-03-11 18:52:49,253] {taskinstance.py:1424} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_01
AIRFLOW_CTX_EXECUTION_DATE=2022-03-10T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-10T00:00:00+00:00
[2022-03-11 18:52:49,255] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-03-11 18:52:49,329] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 3']
[2022-03-11 18:52:49,529] {subprocess.py:85} INFO - Output:
[2022-03-11 18:52:52,560] {subprocess.py:93} INFO - Command exited with return code 0
[2022-03-11 18:52:53,654] {taskinstance.py:1267} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_01, execution_date=20220310T000000, start_date=20220311T185244, end_date=20220311T185253
[2022-03-11 18:52:54,078] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-11 18:52:55,109] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
